DS13/code/modules/EMH/EMH.dm

```diff
@@ -0,0 +1,276 @@
+GLOBAL_VAR_INIT(EMH_present, FALSE) //Stop multiple holograms for now...To avoid mass grief.
+GLOBAL_LIST_INIT(EMH_blacklist, list())
+
+//EMITTER
+
+/obj/machinery/emh_emitter
+	name = "Emergency medical hologram emitter"
+	desc = "Simply click this device and an emergency medical hologram will be summoned. Swipe it with a captain level ID to terminate the current EMH if it's misbehaving."
+	icon = 'DS13/icons/obj/decor/wall_decor.dmi'
+	icon_state = "emh-off"
+	var/mob/living/carbon/human/species/holographic/emh
+	anchored = TRUE
+	density = FALSE
+	pixel_y = 32 //Put on the tile below a wall etc etc modular fun time mapping mayhem
```

Comment:
This should be defined at the instance level, not the type level

Reply:
https://github.com/DDMers/Deep-Space-13/blob/master/code/modules/power/apc.dm#L127